libpod/container_internal.go

```diff
@@ -1290,3 +1293,23 @@ func getExcludedCGroups() (excludes []string) {
 	excludes = []string{"rdma"}
 	return
 }
+
+// namedVolumes returns named volumes for the container
+func (c *Container) namedVolumes() ([]string, error) {
```

Comment:
It's a fundamental problem that we don't have the actual name of the volume in here, just the path it's mounted to in the container

Reply:
Can't fix in this PR, just making a note for myself to go over when I do the DB stuff

Reply:
I agree.  Long range we need to revisit this, especially with the eventual adding of non local Docker volumes.

Reply:
Fixed